cmd/mkroot/commands.go

```diff
@@ -174,125 +148,141 @@ func useLocalInit(repo string) (initPath string, runConfig []byte, err error) {
 	return
 }
 
-func (m *mkfs) Execute() error {
+func (m *mkfs) createLoopDrive() error {
```

Comment:
this one creates an empty loop drive that is used to create the other drives that we need